.github/workflows/prerelease.yaml

```diff
@@ -126,18 +130,9 @@ jobs:
           name: jazzer_releases
           path: _releases/
 
-      - name: read version
-        id: read-version
-        run: |
-          echo ::set-output name=version::\
-          $(sed -nr 's/JAZZER_VERSION = "(.*)"/\1/p' maven.bzl)
-        shell: bash
-
       - name: create release
-        uses: softprops/action-gh-release@v1
+        uses: softprops/action-gh-release@c9b46fe7aad9f02afd89b12450b780f52dacfb2d
```

Comment:
This is just for stability reasons, I guess? Or is there something in the newer commits that we need?